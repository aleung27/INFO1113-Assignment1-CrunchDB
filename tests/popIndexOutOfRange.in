SET z 10 2 93 3
POP Z
POP z
GET z
POP z
GET z
POP z
GET z
POP z
GET z
POP z
GET z
