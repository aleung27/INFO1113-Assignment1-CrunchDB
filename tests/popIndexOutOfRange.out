> ok

> no such key

> 10

> [2 93 3]

> 2

> [93 3]

> 93

> [3]

> 3

> []

> nil

> []

